CMakeLists.txt

```diff
@@ -45,8 +45,8 @@ add_subdirectory(src)
 add_subdirectory(include)
 
 if (VUL_IS_TOP_LEVEL)
-    option(VUL_TESTS "Build tests")
-    if (VUL_TESTS)
+    option(BUILD_TESTS "Build tests")
```

Comment:
Note: Since I continually am updating the update_deps.py logic using the same BUILD_TESTS variable across repos helps quite a bit. Also because this option is only exposed if the project is top level it doesn't negatively impact `add_subdirectory` users.